### ISSUE TITLE
Check to make sure paths are relative to source root dir

### ISSUE THREAD

**Issue by frankhale** [frankhale/toby#79]
Two days ago I switched my development workflow to support one code base and either Electron or NW.js but I may have accidentally screwed up some of the file paths. The file paths should all be relative to the source code directory. I just need to double check to make sure I didn't change anything.

**Comment by frankhale** [comment]
So the path to start the Express server using an external node process is a bit wonky. In my development environment I need it to be like this:

```
node = spawn("node.exe", ["./build/server.js"], { cwd: `${process.cwd()}\\..\\toby` }),
```

NOTE: need to make the path for cwd cross platform by using the path modules separator.

but if running from the normal way you package NW.js or Electron it needs to be:

```
node = spawn("node.exe", ["./build/server.js"], { cwd: process.cwd() }),
```

**Comment by frankhale** [comment]
Closed via 2082958a60fb1d2efa9caeeca49ed817603e6ad6

Working on updating the README with relevant information now
